### ISSUE TITLE
Weekly goal (April 21st - 27th)

### ISSUE THREAD

**Issue by longye-tian** [longye-tian/my-phd#21]
Main Goal

- [ ] construct a program for visualizing healthy lifestyle, i.e., promote visible and immediate gratification for health level feedback
- [ ] Use neural network to approximate the minimal set of DP as IFS

# Math
- [x] Finish Kreyszig Chapter 3

# Computation
- [x] Finish Ken Judd Chapter 2
- [ ] Finish DL code on RBC

# DP, RL and QHD
- [x] Finish reading Jaakkola et al and its proof
- [x] Finish Positive operator on Lp is order continuous
- [ ] Finish Barro 1999
- [ ] Finish the presentation slides for DRL for sustainability
- [ ] Review the literature and proof for Neural network that break the curse of dimensionality. --> Neural Network approximation by R. DeVore, B. Hanin and G. Petrova and Bach 2017.

# Coursework
- [x] Finish ECMT assignment
- [x] Finish DG assignment
- [ ] Finish Fractal wk7,8 lecture
- [ ] Finish assignment 2 for network
- [ ] Finish assignment 3 for fractals
- [x] Week 9 ECMT assignment

## Monday

- [x] Kreyszig section 3.5
- [x] Read Ken Judd Book 2.1-2.6
- [x] QHD reading Barro 1999 - in a relatively simple neoclassical growth model, Barro conjecture a constant rate of discounting is one of the equilibrium. Question: how does this related to existence and uniqueness? See Laibson instantaneous gratification for extension. My idea; partially observable MDP maybe related to QHD
- [x] Jaakkola et al 1994 paper
- [x] Anki
- [x] Exercise
- [x] ECMT assignment

## Tuesday 

- [x] Kreyszig section 3.6
- [x] Read Ken Judd Book finish chapter 2
- [x] Jaakkola cont - Dvoretzky and on Dvorestzky. The key is the proof of theorem 1. We want to extend theorem 1.
- [x] Zaanen Thm 16.2
- [x] QHD cont - two period periodic discounting and q learning Read an intro for POMDP. I think this could related to QHD. -- impatience with lack of perfect forsight. Myopia and impatience.
- [x] Anki
- [ ] Exercise
- [x] DG assignment

## Wednesday

- [x] Kreyszig section 3.7
- [x] DL code 
- [x] Jaakkola cont. - finish the proof for theorem 1of Jaakkola et al.
- [x] QHD cont - periodic discouting is not working. Maybe try not perfect foresight but limited foresight of future state, i.e., noisy transition probability matrix. See Myopia and Impatience
- [x] Zaanen cont. - Thm15.1.i and Example 16.3 Lp is a Banach lattice by proving it has the RF property
- [x] Anki
- [ ] Exercise
- [x] DG assignment cont - Q1, transition and cocycle condition. Check question2 on what is cover on Tensor, Lie bracket calculus. and finish q3

## Thursday

- [x] Kreyszig section 3.8
- [x] DL code - \Xi_torch_init
- [x] Find a theorem that deals with the curse of dimensionality. Start with Neural Network approximation by R. DeVore, B. Hanin and G. Petrova and Bach 2017.
- [x] Jaakkola cont. - go through lemma 2 and lemma 3 and the whole proof assuming lemma 1.
- [x] QHD cont. - think about SIR model Callum Jones. scratch a model that incorporate this and see if there is anything related to QHD.
- [x] Zaanen cont. - first show that Lp norm is order continuous, then positive operator is order bounded hence norm bounded on banach lattice Lp. Use for monotone sequence, norm limit is order limit, we prove that positive operator on Lp is order continuous.  (Note: The proof tree: Beppo Levi Integration of increasing sequence to Lp has the Riesz Fischer property --> Positive operator is order bounded--> Positive operator is norm bounded on Lp--> Lp has order continuous norm, --> positive operator is order continuous on Lp)
- [x] Anki
- [ ] Exercise 
- [x] DP2 reading group meeting 16:00-17:00
- [x] DG assignment cont.- check and submit
- [x] Network assignment 2
- [x] ECMT week 9 assignment

## Friday

- [x] Kreyszig section 3.9
- [x] DL code - \Xi_torch and Resid_torch, train NN
- [x] see if any other similar articles like Bach 2017, see Neural Network approximation by R. DeVore, B. Hanin and G. Petrova; then go into Bach 2017 to see the proof
- [x] Jaakkola cont. - go through the proof using Doob's inequality and supermartingale convergence theorem
- [x] QHD cont.- (skipped) topological equivalence and time transformation, floor function, threshold of noticing difference, i.e., lack of perfect information, history dependent, etc, SIR dynamics.
- [x] Zaanen - continue and finish the proof writing 
- [x] Anki
- [ ] Exercise
- [x] Presentation meeting: 17:00-18:00
- [ ] network assignment - 1.2; 1.3

## Saturday

- [x] Anki
- [x] Kreyszig section3.10 finish exercises and go to section 4.1
- [ ] DL code - train NN on rbc model using full loss function
- [ ] Jaakkola cont. - Go through the proof finish this proof. In both ways, in your way and in the way suggested in the paper. 
- [ ] Curse of dimensionality start preparing the slides: content from neural network computation, focus on what model class can be efficiently approximated, and give reference, especially give the theorem. Dont go through the proof, no time. 
- [ ] Exercise
- [x] Zaanen - proof read and then add why L\infty in not considered, as L\infty norm is not order continuous. 
- [x] DP slides revision 
- [ ] Network assignment- 1.2; 1.3
- [ ] QHD cont.-topological equivalence and time transformation, floor function, threshold of noticing difference, i.e., lack of perfect information, history dependent, etc, SIR dynamics.

## Sunday

- [ ] DL code
- [ ] Jaakkola cont.
- [ ] QHD cont.
- [x] Anki
- [ ] Exercise
- [ ] Network assignment cont.
